crypto/context.c

```diff
@@ -199,14 +199,25 @@ void OSSL_LIB_CTX_free(OSSL_LIB_CTX *ctx)
     OPENSSL_free(ctx);
 }
 
+OSSL_LIB_CTX *OSSL_LIB_CTX_get0_global_default(void)
+{
+#ifndef FIPS_MODULE
```

Comment:
Should the default_context_init RUN_ONCE be called before the return?

Reply:
Yes it should. Good catch - done.